### DISCUSSION TITLE
Any plans to support Microsoft SQL Server?

### DISCUSSION THREAD

**Discussion by ithiru** [keystonejs/keystone#9101] [Questions] [answered]
Team,

I understand that currently keystone supports only Postgres, MySQL and Sqlite. Any plans to support MS SQL Server? 

Curiously, Prisma supports SQL Server, but keystone 6 doesn't. Any particular reason? Is it unsupported?

Thanks
thiru

**Comment by dcousens** [comment, marked as answer]
@ithiru I don't know of any reason why MS SQL couldn't be supported, we are open to pull requests if you, or anyone else, wants to add that support :blue_heart:.

I don't personally have a testing environment for MS SQL - whatever is contributed should update our CI to include MS SQL for testing.
